server/core/src/main/java/dev/slimevr/tracking/processor/skeleton/HumanSkeleton.kt

```diff
@@ -519,6 +523,76 @@ class HumanSkeleton(
 		if (isTrackingRightArmFromController) rightHandTrackerBone.update()
 	}
 
+	private fun updateTrackerYawCorrections() {
+		val trackersToAlign = listOf(
+			headTracker,
+			upperChestTracker,
+			chestTracker,
+			waistTracker,
+			hipTracker,
+		)
+
+		var parentTracker: Tracker? = null
+		for (tracker in trackersToAlign) {
+			if (tracker == null) {
+				continue
+			}
+
+			if (parentTracker == null) {
+				parentTracker = tracker
+				continue
+			}
+
+			if (!tracker.isImu()) {
+				continue
+			}
+
+			updateTrackerYawCorrection(tracker, parentTracker)
+			parentTracker = tracker
+		}
+	}
+
+	private fun isTrackerPointingUp(trackerRot: Quaternion, maxDeviationInRad: Float): Boolean {
+		val trackerUp = trackerRot.sandwich(POS_Y)
+		return trackerUp.angleTo(POS_Y) < maxDeviationInRad
```

Comment:
Surely we have a cleaner way of doing this? It doesn't necessarily matter, but I figure I'd open a discussion on this.

Reply:
This function's going away in the next change. What would be a cleaner way?

Reply:
tbh this is fine